l10n_fi_edicode/data/res_partner_operator_einvoice.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
According to http://verkkolasku.tieke.fi there are 28 operators currently and only 25 are listed. 

- Lexmark is missing
- Netbox is missing
- Palette Software is missing
- Enfo Zender Oy is now Ropo Capital Oy
- TeliaSonera is now Telia Finland Oyj.